src/helpers/subscribe.ts

```diff
@@ -0,0 +1,281 @@
+import { EventEmitter } from "events";
+import asyncGenFromEmit from "@async-generators/from-emitter";
+import { type TablelandTables } from "@tableland/evm";
+import { pollTransactionReceipt } from "../validator/receipt.js";
+import {
+  getTableIdentifier,
+  getContractAndOverrides,
+  type TableIdentifier,
+} from "../registry/contract.js";
+import { extractBaseUrl, type Config } from "../helpers/index.js";
+
+// @ts-expect-error Seems like this package isn't setup to work with modern esm + ts
```

Comment:
What about this one: https://github.com/rolftimmermans/event-iterator
Otherwise, I'm sure its fine?

Reply:
By the way, this is a pretty useful resource: https://github.com/alanshaw/it-awesome

Reply:
I explored using the `event-iterator` package but it seems like it will be a little complicated to wrap an existing node.js EventEmitter instance.  Which feels odd since the description makes it sound like it would be the primary use-case, but all the examples are for streams, or browser events.
The async-generators package works as a one-liner, maybe we stick with that unless an issue arrises?

Reply:
Yeh sure!